test/unit/nuxt.test.js

```diff
@@ -43,4 +43,16 @@ describe('nuxt', () => {
 
     await nuxt.close()
   })
+
+  test('Fail to build when specified plugin isn\'t found', () => {
+    const nuxt = new Nuxt({
```

Comment:
Run a complete build here seems too expensive

Reply:
@clarkdo Not so much because it very quickly fails and exits.